pkg/kv/kvserver/replica_proposal.go

```diff
@@ -655,6 +655,10 @@ func addSSTablePreApply(
 			Size:            sst.BackingFileSize,
 			SmallestUserKey: start.Encode(),
 			LargestUserKey:  end.Encode(),
+
+			// TODO(msbutler): I guess we need to figure out if the backing external
```

Comment:
@itsbilal Given the preso this morning, I guess I should reword this comment to: "we set this true even if the virtual SST's span covers no point keys in the remote sst, to avoid a pebble level panic during ingestion. At download time, if the backing file has no actual keys in the span, the virtual sst will be compacted away and no keys will download.